diff --git a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
index 913bfd9..ce677ac 100644
--- a/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
+++ b/node_modules/@changesets/assemble-release-plan/dist/assemble-release-plan.cjs.dev.js
@@ -67,7 +67,7 @@ function getDependents(releases, packagesByName, dependencyGraph, preInfo) {
         versionRange
       } = getDependencyVersionRange(dependentPackage.packageJson, nextRelease.name); // Firstly we check if it is a peerDependency because if it is, our dependent bump type needs to be major.
 
-      if (depTypes.includes("peerDependencies") && nextRelease.type !== "patch" && (!releases.has(dependent) || releases.has(dependent) && releases.get(dependent).type !== "major")) {
+      if (depTypes.includes("peerDependencies") && nextRelease.type === "major" && (!releases.has(dependent) || releases.has(dependent) && releases.get(dependent).type !== "major")) {
         type = "major";
       } else {
         if ( // TODO validate this - I don't think it's right anymore
