diff --git a/node_modules/svelte-preprocess/dist/transformers/babel.js b/node_modules/svelte-preprocess/dist/transformers/babel.js
index 4e59b2d..50f47ff 100644
--- a/node_modules/svelte-preprocess/dist/transformers/babel.js
+++ b/node_modules/svelte-preprocess/dist/transformers/babel.js
@@ -13,6 +13,15 @@ const transformer = async ({ content, filename, options, map = undefined, }) =>
         minified: false,
         ast: false,
         code: true,
+        // https://github.com/sveltejs/svelte-preprocess/pull/449
+        caller: {
+            name: 'svelte-preprocess',
+            supportsStaticESM: true,
+            supportsDynamicImport: true,
+            supportsTopLevelAwait: true,
+            supportsExportNamespaceFrom: true,
+            ...(options && options.caller),
+        },
     };
     const { code, map: sourcemap } = await core_1.transformAsync(content, babelOptions);
     return {
